trade-finance-manager-api/src/v1/__mocks__/mock-utilisation-report.ts

```diff
@@ -8,14 +8,20 @@ const MOCK_AZURE_FILE_INFO = {
   mimetype: 'text/csv',
 };
 
+const MOCK_REPORT_PERIOD = {
+  startMonth: 11,
+  startYear: 2023,
+  endMonth: 11,
+  endYear: 2023,
+}
```

Comment:
Should this be
```
const MOCK_REPORT_PERIOD = {
  start: { month: 11, year: 2023 },
  end: { month: 11, year: 2023 },
}
```
?

Reply:
I think you should add the type explicitly to `MOCK_REPORT_PERIOD` so that this error in structure gets flagged up for you :)